Dockerfile

```diff
@@ -0,0 +1,72 @@
+ARG REGISTRY_PREFIX=''
```

Comment:
Note that all "ARG"s can be provided using dockers --build-arg command line option.
When no option is provided, a reasonable default is used.